casanovo/casanovo.py

```diff
@@ -1,13 +1,17 @@
 """The command line entry point for Casanovo."""
 
 import datetime
+import email.utils
 import functools
+import hashlib
 import logging
+import urllib
```

Comment:
**todo:** I don't think this is necessary (rather the relevant import is `urllib.parse` a few lines further down).